### ISSUE TITLE
Fallback images not being captured

### ISSUE THREAD

**Issue by alykat** [nprapps/dailygraphics-next#10]
Oddly, the fallback image capture feature stopped working for me today. I've tried rebooting, using different browsers (Chrome, Firefox and Safari/Mac - High Sierra), trying different graphics — all with the same result.

The error in my browser console (replacing the actual graphic slug with `__SLUG__`):

```
POST http://localhost:7777/graphic/__SLUG__/captureFallback 500 (Internal Server Error)
(anonymous) @ parentPage.js:30
```

Which points to this line in the code:

```
var capture = await fetch("./captureFallback", { method: "POST" });
```

In my local webserver, I see this sequence:

```
Trying capture...
Carlo window ready
Loading:  http://localhost:7777/graphic/__SLUG__/index.html
{ TimeoutError: Navigation Timeout Exceeded: 3000ms exceeded
    at Promise.then (/Users/ahurt/src/dailygraphics-next/node_modules/puppeteer-core/lib/FrameManager.js:1276:21)
  -- ASYNC --
    at Frame.<anonymous> (/Users/ahurt/src/dailygraphics-next/node_modules/puppeteer-core/lib/helper.js:144:27)
    at Page.goto (/Users/ahurt/src/dailygraphics-next/node_modules/puppeteer-core/lib/Page.js:624:49)
    at Page.<anonymous> (/Users/ahurt/src/dailygraphics-next/node_modules/puppeteer-core/lib/helper.js:145:23)
    at Object.snapGraphic (/Users/ahurt/src/dailygraphics-next/lib/puppetry.js:20:14)
    at process._tickCallback (internal/process/next_tick.js:68:7) name: 'TimeoutError' }
```

(And then after that it continues with the deploy.)

**Comment by thomaswilburn** [comment]
Okay, looks like the page is taking a long time to load for some reason. We can increase the timeout, but I'll want to figure out why it's hitting that in the first place at some point. Will test.

(Changing browsers won't matter, it always uses your system Chrome to capture. Which could also be the issue, if Chrome updated and is lagging for some reason.)

**Comment by thomaswilburn** [comment]
Hm, I'm not able to duplicate. Let's make some time when I'm in DC to try this on your machine, if it continues.

**Comment by alykat** [comment]
Addressed in 964017d343fd6789f307b241bcd25e349d96a26b, e8cc6faa4b5b4c47f775b0a733483e489637fedc, bfbc69886e09556e675e0c084c14e7d17baf2316, 5501d539fa67165afe245f55adf38f37ce2e135c, 6a866cb1ad5ee525c69c3652b65df98b85ac220c
